src/main/java/icbm/classic/mods/cubicchunks/CubicChunks.java

```diff
@@ -0,0 +1,60 @@
+package icbm.classic.mods.cubicchunks;
+
+import icbm.classic.ICBMClassic;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.Loader;
+
+import java.lang.reflect.Method;
+
+public class CubicChunks {
+
+    public static CubicChunks INSTANCE = new CubicChunks();
+    boolean present = false;
+
+    Method getMinHeightMethod;
+    Method getMaxHeightMethod;
+
+    @SuppressWarnings("JavaReflectionMemberAccess")
+    private CubicChunks() {
+        ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Looking for CubicChunks");
+        if (!Loader.isModLoaded("cubicchunks")) {
+            ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Didn't find CubicChunks");
+            return;
+        }
+
+        ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Found CubicChunks");
+        try {
+            getMinHeightMethod = World.class.getMethod("getMinHeight");
+            getMaxHeightMethod = World.class.getMethod("getMaxHeight");
+            present = true;
+        } catch (Exception e) {
+            ICBMClassic.logger().error("ICBMClassic>>CubicChunks: Failed to hook CubicChunks");
+            e.printStackTrace();
```

Comment:
This should be broken down into each predictable exception so we can add more information.

Reply:
Can you elaborate here?, the api is pretty solid (no changed in like 3y) i don't really expect this to happen

Reply:
He means that you should explicitly handle each exception that `getMethod` can throw (NoSuchMethodException, NullPointerException, and SecurityException) so if anything happens for whatever reason, we know exactly why it did.